tests/components/geonetnz_quakes/test_config_flow.py

```diff
@@ -22,46 +15,24 @@
     CONF_UNIT_SYSTEM,
 )
 
-from tests.common import MockConfigEntry
-
-
-@pytest.fixture
-def config_entry():
-    """Create a mock GeoNet NZ Quakes config entry."""
-    return MockConfigEntry(
-        domain=DOMAIN,
-        data={
-            CONF_LATITUDE: -41.2,
-            CONF_LONGITUDE: 174.7,
-            CONF_RADIUS: 25,
-            CONF_UNIT_SYSTEM: "metric",
-            CONF_SCAN_INTERVAL: 300.0,
-            CONF_MMI: 4,
-            CONF_MINIMUM_MAGNITUDE: 0.0,
-        },
-        title="-41.2, 174.7",
-    )
-
 
 async def test_duplicate_error(hass, config_entry):
     """Test that errors are shown when duplicates are added."""
     conf = {CONF_LATITUDE: -41.2, CONF_LONGITUDE: 174.7, CONF_RADIUS: 25}
-
     config_entry.add_to_hass(hass)
-    flow = config_flow.GeonetnzQuakesFlowHandler()
-    flow.hass = hass
 
-    result = await flow.async_step_user(user_input=conf)
-    assert result["errors"] == {"base": "identifier_exists"}
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": "user"}, data=conf
+    )
+    assert result["type"] == data_entry_flow.RESULT_TYPE_ABORT
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5122969)